src/components/Popups/PopupCursor.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import { FormattedMessage } from 'react-intl';
+import messages from './messages';
+import PopupBase from './PopupBase';
+import { Options } from './Popper';
+import './PopupCursor.scss';
+
+export type Props = {
+    isActive: boolean;
+};
+
+const options: Partial<Options> = {
+    modifiers: [
+        {
+            name: 'offset',
+            options: {
+                offset: [0, 20],
+            },
+        },
+    ],
+    placement: 'right',
+};
+
+const generateGetBoundingClientRect = (x = 0, y = 0) => () => ({
+    width: 0,
```

Comment:
Would a width and height of 0 interfere with Popper's ability to position it based on viewport boundaries?

Reply:
> Would a width and height of 0 interfere with Popper's ability to position it based on viewport boundaries?

It works ok in manual testing. I actually copied this function from Popper.js example page 😂